src/config.rs

```diff
@@ -100,7 +104,7 @@ pub struct Discv5Config {
     /// The max limit for peers with unreachable ENRs. Benevolent examples of such peers are peers
     /// that are discovering their externally reachable socket, nodes must assist at least one
     /// such peer in discovering their reachable socket via ip voting, and peers behind symmetric
-    /// NAT. Default is no limit. Minimum is 1.
+    /// NAT. Default is no limit. Minimum is 10.
```

Comment:
nice, that crosses off the second todo I still had open https://github.com/sigp/discv5/pull/176#issuecomment-1877505234